src/core/lib/gpr/fork.cc

```diff
@@ -38,18 +38,25 @@ void grpc_fork_support_init() {
   fork_support_enabled = 1;
 #else
   fork_support_enabled = 0;
+#endif
   char* env = gpr_getenv("GRPC_ENABLE_FORK_SUPPORT");
   if (env != nullptr) {
     static const char* truthy[] = {"yes",  "Yes",  "YES", "true",
                                    "True", "TRUE", "1"};
+    static const char* falsey[] = {"no",    "No",    "NO", "false",
+                                   "False", "FALSE", "0"};
     for (size_t i = 0; i < GPR_ARRAY_SIZE(truthy); i++) {
       if (0 == strcmp(env, truthy[i])) {
         fork_support_enabled = 1;
       }
     }
+    for (size_t i = 0; i < GPR_ARRAY_SIZE(falsey); i++) {
```

Comment:
Could you make a change such that this loop is skipped if env variable was a truthy value?

Also, put a "break" statement to get out of the loop (here and the truthy loop) once fork_support_enabled is set

Reply:
Fixed